.github/workflows/haskell-ci.yml

```diff
@@ -0,0 +1,108 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  cabal:
+    name: cabal ${{ matrix.os }} / ghc ${{ matrix.ghc }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - macOS-latest
```

Comment:
I think that even macos will fail. it also doesn't have native containers support. You would need to install hypervisor.

Reply:
Alright. Are we ok with ignoring mac and windows in CI?
I'd say it's fine as this package doesn't have any native dependencies and also the previous travis CI didn't test these platforms.

Reply:
I think there is nothing OS specific in this lib. I don't think we need to test on multiple platforms.